SU2_CFD/include/driver_structure.hpp

```diff
@@ -413,63 +413,60 @@ class CMultiZoneDriver : public CDriver {
 
 /*!
  * \class CSpectralDriver
- * \brief Class for driving an iteration of a spectral method problem using multiple zones.
+ * \brief Class for driving an iteration of harmonic balance method problem using multiple zones.
```

Comment:
Perhaps we should also take this chance to rename the child driver class to CHarmonicBalanceDriver or CHBDriver? This would be more consistent with the replacements of "time spectral" with "harmonic balance" throughout the rest of the code.